common/djangoapps/edxmako/middleware.py

```diff
@@ -62,4 +70,6 @@ def get_template_request_context():
     for processor in get_template_context_processors():
         context.update(processor(request))
 
+    request_cache_dict[cache_key] = context
```

Comment:
Why the mako caching here?  Does this not already implicitly happen with the request cache middleware? (I don't really know what edxmako is, but this strikes me as a particularly pointed addition of caching.)